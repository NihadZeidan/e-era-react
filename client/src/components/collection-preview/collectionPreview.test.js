import { shallow } from "enzyme";
import CollectionPreview from "./collectionPreview.component";

it("Should render CollectionPreview component", () => {
  const mockTitle = "test title";
  const mockItems = [
    { id: 1, name: "test name", imageUrl: "test imageUrl", price: 10 },
  ];
  expect(
    shallow(<CollectionPreview title={mockTitle} items={mockItems} />)
  ).toMatchSnapshot();
});
