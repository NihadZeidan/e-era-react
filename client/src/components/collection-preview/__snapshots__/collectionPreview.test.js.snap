// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render CollectionPreview component 1`] = `ShallowWrapper {}`;
